infra/cifuzz/actions/build_fuzzers/action.yml

```diff
@@ -5,17 +5,12 @@ inputs:
   project-name:
     description: 'Name of the corresponding OSS-Fuzz project.'
     required: true
-  fuzz-seconds:
-    description: 'The total time allotted for fuzzing in seconds.'
-    required: true
-    default: 360
   dry-run:
     description: 'If set, run the action without actually reporting a failure.'
-    default: false
+    default: 0
```

Comment:
Why was this change made? To make it easier to convert to a Python bool? Changing the type of an input variable to something that doesn't fit as well seems like the wrong approach here.

Reply:
@jonathanmetzman requested it here https://github.com/google/oss-fuzz/pull/3336#discussion_r375438071  what are the thoughts on the correct way to proceed?

Reply:
I didn't really know what went into that environment variable when I asked for this.
@Leo-Neat So what was the value of this environment variable when this was "false", was the value "false", I think that's a bit uglier than using 0, but maybe it's worth doing since this is user facing.

Reply:
I am going to keep true and false to help the user understanding.